akka-cluster/src/multi-jvm/scala/akka/cluster/artery/ArterySunnyWeatherSpec.scala

```diff
@@ -24,12 +25,25 @@ object SunnyWeatherMultiJvmSpec extends MultiNodeConfig {
   // Note that this test uses default configuration,
   // not MultiNodeClusterSpec.clusterConfig
   commonConfig(ConfigFactory.parseString("""
-    akka.actor.provider = akka.cluster.ClusterActorRefProvider
-    akka.loggers = ["akka.testkit.TestEventListener"]
-    akka.loglevel = INFO
-    akka.remote.log-remote-lifecycle-events = off
-    akka.cluster.failure-detector.monitored-by-nr-of-members = 3
+    akka {
+      actor {
+        provider = akka.cluster.ClusterActorRefProvider
+        serialize-creators = false
+        serialize-messages = false
```

Comment:
that should not be needed, I hope
I think I disabled that for performance tests, for performance reasons.